plain-js/main.js

```diff
@@ -416,10 +416,43 @@ window.onload = async () => {
     };
   }
 
+  const cameraSwitchButtons = document.getElementsByClassName("camera-switch-button");
+
+  for (let i = 0; i < cameraSwitchButtons.length; i++) {
+    const button = cameraSwitchButtons[i];
+    button.onclick = async (e) => {
+      const controller =
+        e.target.parentElement.parentElement.parentElement.className;
```

Comment:
Is this safe? Maybe there's a way to retrieve the same element without relying on the grand-grand-parents